uber/templates/jobs/form.html

```diff
@@ -23,6 +23,32 @@
         });
         $("[name=restricted],[name=extra15]").attr("checked", false);
     }
+    $().ready(function() {
+        {% if not job.is_new %}
```

Comment:
This looks good, but can you move this if check to outside the `ready` function? That way, if the job `is_new`, we won't have just a random empty `$().ready(function () { });` function rendered in the HTML.